charts/chronicle/templates/opa-init.yaml

```
@@ -0,0 +1,78 @@
+{{- if .Values.opa.enabled }}
+{{$stlServiceName := include "lib.call-nested" (list . "sawtooth" "common.names.fullname")}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-opa-init
+  labels: {{ include "chronicle.labels" . | nindent 4 }}
+    component: opa-init
+spec:
+  backoffLimit: 5
+  template:
+    metadata:
+      labels: {{- include "chronicle.labels" . | nindent 8 }}
+        component: opa-init
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ include "lib.serviceAccountName" . }}
+      automountServiceAccountToken: true
+      affinity: {{ include "lib.safeToYaml" .Values.affinity | nindent 8 }}
+      containers:
+        - name: opa-init
+          {{- include "lib.image" (dict "imageRoot" .Values.opa.opaInit.image "global" .Values.global ) | nindent 10 }}
+          command: [ "bash", "-c"]
+          args:
+            - -exc
+            - |
+              HOST={{ $stlServiceName }}.{{ .Release.Namespace }}.svc.cluster.local
+              PORT={{ include "chronicle.sawtooth.sawcomp" . }}
+
+              /script/wait-for-it.sh $HOST:$PORT --timeout=0
+
+              cd /tmp
+              /usr/local/bin/opactl generate --output ./root.pem
```

```suggestion
              /usr/local/bin/opactl generate --output root.pem
```
I'd think the various `./` here are no-ops?

---

Do you mean you can just leave them out? They're just copied over from the Docker compose setup.

---

Yes, you should be able to just leave them out. Not a blocker for this PR.